src/PowerShell/Microsoft.WinGet.Client/PSObjects/InstalledCatalogPackage.cs

```diff
@@ -0,0 +1,89 @@
+﻿// -----------------------------------------------------------------------------
+// <copyright file="InstalledCatalogPackage.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Client.PSObjects
+{
+    using System.Linq;
+
+    /// <summary>
+    /// CatalogPackage wrapper object for displaying to PowerShell.
+    /// </summary>
+    public class InstalledCatalogPackage

```

Comment:
The only difference is whether we use InstalledVersion or DefaultInstallVersion to obtain the Version that we display to the user. I created a parent class called ```CatalogPackage``` that ```FoundCatalogPackage``` and ```InstalledCatalogPackage``` inherit from so that there less repeated properties.